src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplayVisitor.vb

```diff
@@ -315,6 +315,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
         End Sub
 
+        Friend Overrides Sub VisitPreprocessing(symbol As IPreprocessingSymbol)
```

Comment:
This change should be backed by a test under Compilers

Reply:
Are symbol visitors tested directly in VB? I can't find anything like what is done in C#

Reply:
This comment refers to symbol display changes.